### PR TITLE
[Bug 1117204] Add a Thunderbird download button to product page

### PR DESCRIPTION
**Pull request by safwanrahman**
This will add a Download Button of Thunderbird in the Thunderbird product page
r?

### CODE REVIEW

**Comment by BReduardokramer** [comment]
I do it. Thanks.
Em 06/01/2015 18:03, "Safwan Rahman" <notifications@github.com> escreveu:

**Comment by safwanrahman** [comment]
This will make a Button like this in the Download Page
firefox_screenshot_2015-01-06t19-39-23 455z

File: kitsune/products/templates/products/product.html [hunk 1]
```diff
@@ -52,6 +52,26 @@ <h1 class="product-title cf">
         </ul>
       </div>
     {% endif %}
+
+    {% if product.slug == 'thunderbird' %}
+      <div class="download-thunderbird">
+        <a href="http://www.mozilla.org/thunderbird" class="download-button"
+         data-ga-click="_trackEvent | Download Button | Panel Opening"
```
Comment (rlr) [inline kitsune/products/templates/products/product.html]: Can you change "Panel Opening" for "Download Thunderbird"?

File: kitsune/products/templates/products/product.html [hunk 2]
```diff
@@ -52,6 +52,26 @@ <h1 class="product-title cf">
...
+         data-latest-version="{{ latest_version }}">
```
Comment (rlr) [inline kitsune/products/templates/products/product.html]: you can remove this attribute. This was for Firefox stuff.

File: kitsune/products/templates/products/product.html [hunk 3]
```diff
@@ -52,6 +52,26 @@ <h1 class="product-title cf">
...
+          <span class="download-content">
+            <strong class="download-title">{{ _('Thunderbird') }}</strong>
+            <span class="download-subtitle">{{ _('Free Download') }}</span>
+            <span class="download-lang">{{ _('English (US)') }}</span>
+          </span>
+        </a>
+        <ul class="info">
+          <li><a href="http://www.mozilla.org/thunderbird/all.html">{{ _('Systems and Languages') }}</a></li>
+          <li><a href="http://www.mozilla.org/thunderbird/releases">{{ _("Release Notes") }}</a></li>
```
Comment (rlr) [inline kitsune/products/templates/products/product.html]: Use single quotes please.

File: kitsune/sumo/static/less/products.less [hunk 4]
```diff
@@ -278,8 +278,132 @@
...
     margin: 0 0 0 8px;
   }
 }
+
+.logo-sprite {
+  float: left;
+  margin: 0 8px 0 0;
+}
+
+.download-thunderbird {
```
Comment (rlr) [inline kitsune/sumo/static/less/products.less]: did you copy paste all these styles from somewhere?

Comment (safwanrahman) [inline kitsune/sumo/static/less/products.less]: Yes. @rlr. I forgot to remove something ;)
Now its Ok?

**Comment by rlr** [comment]
I am curious where you got the styles from? I'm assuming it's from some styleguide somewhere or something?

**Comment by safwanrahman** [comment]
@rlr Actually I followed the code of Firefox Download Button given in that style sheet and customize that Button for Thunderbird :)

**Comment by safwanrahman** [comment]
@rlr r?

**Comment by rlr** [comment]
So, the little links are very hard to click. If you try to hover over them, notice that something is blocking on top of them. It is probably a z-index thing.

**Comment by rlr** [comment]
Also, I can't tell if that dotted line should be there or not. It looks out of place.

**Comment by safwanrahman** [comment]
@rlr I have made the Button shorter so it will not block the things.
Also the bottom Dotted line is removed :)
now r?

**Comment by mythmon** [comment]
@safwanrahman, it would be a lot easier to review things if you didn't squash your commits. After review has started, please just add new commits to the branch, with a commit message that explains what changed just in that commit. It makes it a lot easier to do this back and forth with code review.

**Comment by safwanrahman** [comment]
@mythmon, Ok I understand! Will take care of this in future. :)

File: kitsune/products/templates/products/product.html [hunk 5]
```diff
@@ -52,6 +52,26 @@ <h1 class="product-title cf">
         </ul>
       </div>
     {% endif %}
+
+    {% if product.slug == 'thunderbird' %}
+      <div class="download-thunderbird">
+        <a href="http://www.mozilla.org/thunderbird" class="download-button"
+         data-ga-click="_trackEvent | Download Button | Download Thunderbird"
+         data-latest-version="{{ latest_version }}">
```
Comment (rlr) [inline kitsune/products/templates/products/product.html]: this (data-latest-version) you can remove, like I mentioned before

**Comment by safwanrahman** [comment]
@rlr Now it seems OK?

**Comment by rlr** [comment]
Looks good. Thank you!

**Comment by rlr** [comment]
77a41519e4254e6ada71b027c6b40a3b8c8ef3d5 [Bug 1117204] Add a Thunderbird download button to product page

**Pull request closed without merging**